app/models/label/LabelTable.scala

```diff
@@ -464,6 +464,14 @@ object LabelTable {
         "LEFT JOIN ( SELECT NULL AS validation_result ) AS user_validation ON lb.label_id = NULL"
       }
 
+    // Query to get comments and make them into a list
+    val commentsJoin: String =
+      s"""LEFT JOIN (
+         |    SELECT label_id, COALESCE(string_agg(COALESCE(comment, ''), ':'), '') AS comments
```

Comment:
Since comment is an option, it can be null. The comment being null will result in it displaying `None` on the frontend.

Reply:
Well the comment can be null because you left join with the table with the comments. But _within_ the `validation_task_comment` table, comments aren't null. And this is before you're doing the left join, so at this point comments won't be null!

Reply:
That makes sense. I was thinking about comments being null on the frontend and not in the DB. Just updated.